Something went wrong when trying to run rari server (yarn run start)

When I tried to run the web server of MDN (`yarn run start`), it failed because the root directory of the process contains spaces...

```
[rari] /bin/sh: 1: /sdb1/Git: Permission denied
[rari] /sdb1/Git Repositories/Markdown/mdn@content/node_modules/@mdn/rari/bin/rari serve -vv exited with code 126
--> Sending SIGTERM to other processes..
[server] node:internal/modules/cjs/loader:1147
[server]   throw err;
[server]   ^
[server] 
[server] Error: Cannot find module '/sdb1/Git'
[server]     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
[server]     at Module._load (node:internal/modules/cjs/loader:985:27)
[server]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
[server]     at node:internal/main/run_main_module:28:49 {
[server]   code: 'MODULE_NOT_FOUND',
[server]   requireStack: []
[server] }
[server] 
[server] Node.js v21.2.0
[server] node /sdb1/Git Repositories/Markdown/mdn@content/node_modules/@mdn/yari/server/index.js exited with code 1
All tasks completed successfully.
```

It'd be great if you added support for such an uncommon case....

--- a/bins/server.mjs
+++ b/bins/server.mjs
@@ -21,7 +21,7 @@ const { commands, result } = concurrently(
       prefixColor: "red",
     },
     {
-      command: `${rariBin} serve -vv`,
+      command: `"${rariBin}" serve -vv`,
       name: "rari",
       prefixColor: "blue",
       env: {

--- a/server/cli.ts
+++ b/server/cli.ts
@@ -21,7 +21,7 @@ const { commands, result } = concurrently(
       prefixColor: "red",
     },
     {
-      command: `${rariBin} serve -vv`,
+      command: `"${rariBin}" serve -vv`,
       name: "rari",
       prefixColor: "blue",
       env: {

